code/.eslintrc.json

```diff
@@ -1,23 +0,0 @@
-{
```

Comment:
This file should not be deleted otherwise your build will fail

Reply:
That'll do it. Not sure why that happened